project/Http4sPlugin.scala

```diff
@@ -219,14 +218,14 @@ object Http4sPlugin extends AutoPlugin {
     extractApiVersion(version).productIterator.mkString("/v", ".", "")
 
   /**
-   * @return the version we want to document, for example in tuts,
-   * given the version being built.
-   *
-   * For snapshots after a stable release, return the previous stable
-   * release.  For snapshots of 0.16.0 and 0.17.0, return the latest
-   * milestone.  Otherwise, just return the current version.  Favors
-   * scalaz-7.2 "a" versions for 0.15.x and 0.16.x.
-   */
+    * @return the version we want to document, for example in tuts,
+    * given the version being built.
+    *
+    * For snapshots after a stable release, return the previous stable
+    * release.  For snapshots of 0.16.0 and 0.17.0, return the latest
+    * milestone.  Otherwise, just return the current version.  Favors
+    * scalaz-7.2 "a" versions for 0.15.x and 0.16.x.
+    */
```

Comment:
wat is dis

Reply:
An artifact of an emacs minor mode that doesn't play well with scala-mode.

Reply:
aha